packages/core/src/components/tabs/test/tabs.ct.ts

```diff
@@ -64,6 +64,28 @@ test('should not change tab by tab click event', async ({ mount, page }) => {
   await expect(lastTab).not.toHaveClass(/selected/);
 });
 
+test('should not change tab by native click event on prevent default', async ({
+  mount,
+  page,
+}) => {
+  await mount(`
+    <ix-tabs>
+      <ix-tab-item onClick={(event) => event.preventDefault()}>Item 1</ix-tab-item>
+      <ix-tab-item onClick={(event) => event.preventDefault()}>Item 2</ix-tab-item>
+      <ix-tab-item onClick={(event) => event.preventDefault()}>Item 3</ix-tab-item>
+    </ix-tabs>
+  `);
+  const tabs = page.locator('ix-tabs');
+  const firstTab = page.locator('ix-tab-item').nth(0);
+  const lastTab = page.locator('ix-tab-item').nth(2);
+
+  await lastTab.click();
+
+  await expect(tabs).toHaveClass(/hydrated/);
+  await expect(firstTab).toHaveClass(/selected/);
```

Comment:
First tab will be selected per default.
Maybe select tab 2 and then click tab 3.
Then expect that tab 2 is still selected.